Directory.Build.props

```diff
@@ -54,7 +54,13 @@
     <ManPagesDir>$([MSBuild]::NormalizeDirectory('$(DocsDir)', 'manpages'))</ManPagesDir>
     <CoreLibSharedDir>$([MSBuild]::NormalizeDirectory('$(LibrariesProjectRoot)', 'System.Private.CoreLib', 'src'))</CoreLibSharedDir>
   </PropertyGroup>
-
+  
+  <!-- SourceLink -->
+  <PropertyGroup>
+    <!-- This is needed to embed build-time generated sources such as eventing and resource files to sourcelink PDBs. -->
+    <EmbedUntrackedSources>true</EmbedUntrackedSources>
```

Comment:
Should this be in `Versions.props`? We usually enable/disable features from arcade there. @tmat ?

Reply:
Sourcelink lives in a separate repo. https://github.com/dotnet/sourcelink. We can still move to ```versions.props``` if we want.

Reply:
Oh, cool. Then this is fine here.

Reply:
It shouldn't be necessary here. It's already set by Arcade here: [https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L18](https://github.com/dotnet/arcade/blob/master/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectDefaults.props#L18). Is this props file not imported for some reason?

---
In reply to: [391824522](https://github.com/dotnet/runtime/pull/33521#discussion_r391824522) [](ancestors = 391824522)

Reply:
It should be imported.